databases/week3/homework3.txt

```diff
@@ -0,0 +1,159 @@
+Use mealsharing;
+
+CREATE TABLE `mealsharing`.`meal` (
+  `id` INT(10) NULL AUTO_INCREMENT,
+  `title` VARCHAR(45) NOT NULL,
+  `description` VARCHAR(255) NOT NULL,
+  `location` VARCHAR(255) NULL,
+  `when` DATETIME NOT NULL,
+  `Max_reservation` INT NULL,
+  `price` DECIMAL(10,2) NULL,
+  `created_date` DATE NULL,
+  PRIMARY KEY (`id`));
+
+CREATE TABLE `mealsharing`.`reservation` (
+  `id` INT (10)NOT NULL,
+  `number_of_guests` INT NOT NULL,
+  `meal_id` INT NOT NULL,
+  `created_date` DATE NOT NULL,
+  PRIMARY KEY (`id`));
+
+CREATE TABLE `mealsharing`.`review` (
+  `id` (10)INT NOT NULL,
+  `title` VARCHAR(45) NOT NULL,
+  `desciption` TEXT NOT NULL,
+  `meal_id` INT NOT NULL,
+  `stars` INT NULL,
+  `created_date` DATE NOT NULL,
+  PRIMARY KEY (`id`));
+
+ALTER TABLE `mealsharing`.`review` 
+CHANGE COLUMN `stars` `stars` INT NULL ;
```

Comment:
You don't need to add this alter table statement. Just make sure all the create table statements are up-to-date before you commit the file to git